extensions/VisualEditor/modules/ve-mw/ui/tools/ve.ui.MWTransclusionDialogTool.js

```diff
@@ -54,6 +54,11 @@ ve.ui.MWTransclusionDialogTool.static.template = null;
 ve.ui.MWTransclusionDialogTool.static.isCompatibleWith = function ( model ) {
 	var compatible;
 
+	//if wikiaInfoboxTransclusionBlock use ve.ui.WikiaInfoboxDialogTool instead
+	if (model.type === 'wikiaInfoboxTransclusionBlock') {
```

Comment:
is there a possibility to use `ve.ce.WikiaInfoboxTransclusionBlockNode.static.name` instead of hardcoded value?